apps/cow-fi/app/actions.ts

```diff
@@ -0,0 +1,27 @@
+'use server'
+
+import { searchArticles as searchArticlesService } from '../services/cms'
+
+/**
+ * Server action to search for articles
+ */
+export async function searchArticlesAction(searchTerm: string, page: number = 0, pageSize: number = 100) {
+  console.log('Server action called with:', { searchTerm, page, pageSize })
+  try {
+    const results = await searchArticlesService({ searchTerm, page, pageSize })
+    console.log('Server action results:', {
```

Comment:
Please remove the log before merging the PR, otherwise it will add additional costs in Vercel :)